cheatsheets/Authentication_Cheat_Sheet.md

```diff
@@ -110,14 +110,14 @@ Incorrectly implemented error messages in the case of authentication functionali
 
 #### Authentication Responses
 
-Whatever the authentication feature involved (login, password reset or password recovery) an application must respond with a generic error message regardless of whether:
+Using any of the authentication mechanisms (login, password reset or password recovery) an application must respond with a generic error message regardless of whether:
 * The user ID or password was incorrect.
 * The account does not exist.
 * The account is locked or disabled.
 
-The account creation feature is also impacted and the same approach of generic error message can be applied regarding the case in which the user exists.
+The account registration feature should also be taken into consideration, and the same approach of generic error message can be applied regarding the case in which the user exists.
 
-The objective is to prevent to create a [discrepancy factor](https://cwe.mitre.org/data/definitions/204.html) allowing an attacker to mount a user enumeration action against the feature.
+The objective is to prevent to create a [discrepancy factor](https://cwe.mitre.org/data/definitions/204.html) allowing an attacker to mount a user enumeration action against the application.
```

Comment:
replace `to create a` with `the creation of a`

Reply:
Fix applied in coming commit.